projects/aqlprofile/src/core/tests/CMakeLists.txt

```diff
@@ -25,7 +25,12 @@ SET(AQLPROFILE_MEMORYMANAGER_SOURCES
 )
 
 target_sources(gfx9-memory-manager-test PRIVATE ${AQLPROFILE_MEMORYMANAGER_SOURCES})
-target_include_directories(gfx9-memory-manager-test PRIVATE ${CMAKE_CURRENT_SOURCE_DIR} ${LIB_DIR} ${LIB_DIR}/core/include)
+target_include_directories(gfx9-memory-manager-test
+    PRIVATE
+        ${CMAKE_CURRENT_SOURCE_DIR}
+        ${LIB_DIR}
+        ${LIB_DIR}/core/include
+        $<TARGET_PROPERTY:aqlprofile::aqlprofile-lib,INCLUDE_DIRECTORIES>)
```

Comment:
Frankly, a aqlprofile-headers target is preferable

Reply:
Done